examples/styling_visualization/lineage_commands.py

```diff
@@ -0,0 +1,253 @@
+"""
```

Comment:
did you mean to include this file?

Reply:
nope! Just copied the full lineage example. Will remove